create-publication-from-scopus/src/test/java/no/sikt/nva/scopus/PublicationPersistenceServiceTest.java

```diff
@@ -0,0 +1,12 @@
+package no.sikt.nva.scopus;
+
+import org.junit.jupiter.api.Test;
+
+
+class PublicationPersistenceServiceTest {
+
+    @Test
+    public void testNothing() {
```

Comment:
Should likely be tested by handler

Reply:
It is just a placeholder since the functionality will be implemented later. And we had to give JaCoco a bone to chew on.

Reply:
We will follow your test premisses.